Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -60,8 +65,7 @@ Build the run_limma function
 ```{r}
 
 run_limma <- function(data_to_model,
```

Comment:
Updated the `run_limma` function to split it into two functions, where the first one returns the limma fit object, since we want this later when running `DEqMS`.